parsing/ast_helper.mli

```diff
@@ -73,6 +73,8 @@ module Typ :
     val extension: ?loc:loc -> ?attrs:attrs -> extension -> core_type
 
     val force_poly: core_type -> core_type
+
+    val varify_constructors: str list -> core_type -> core_type
```

Comment:
Hi @objmagic :-) The function was added by Jacques Le Normand in 2010: 90a6348480417e7f769390abb258fb3dfd5a2a1c